static/index2.html

```diff
@@ -2,8 +2,8 @@
 <head>
     <title>Bank Connect</title>
     <link type="text/css" rel="stylesheet" href="style.css">
-    <link type="text/css" rel="preload" href="dropzone.css" as="style" onload="this.rel='stylesheet'">
```

Comment:
Specifying rel="preload" is still not available on "Firefox" and "Firefox for Android", and we are not sure of performance impact, I guess we can ignore this in future for sake of clarity? I guess this is just recommendation from dropzone site.